entities/weapons/ix_hands.lua

```diff
@@ -427,7 +427,7 @@ function SWEP:PrimaryAttack()
 							damageInfo:SetAttacker(self:GetOwner())
 							damageInfo:SetInflictor(self)
 							damageInfo:SetDamage(damage)
-							damageInfo:SetDamageType(DMG_SLASH)
+							damageInfo:SetDamageType(DMG_GENERIC)
```

Comment:
According to the gmod wiki the fists should use "DMG_GENERIC". DMG_SLASH is reserved to the Manhacks, Antlions, Antlion Guards, etc.